feeadjuster/feeadjuster.py

```diff
@@ -83,6 +80,7 @@ def maybe_adjust_fees(plugin: Plugin, scids: list):
                 and abs(last_percentage - percentage) < update_threshold):
             continue
 
+        assert 0 <= percentage and percentage <= 1
```

Comment:
The first part of the check hast the wrong direction. Should be `>=` and not `<=`

;)

Reply:
Check it again! ;)

Reply:
lol

Reply:
one should never review code on a mobile phone ;)